gateway/ln-gateway/src/client.rs

```diff
@@ -112,47 +61,59 @@ dyn_newtype_define! {
 #[derive(Debug, Clone)]
 pub struct StandardGatewayClientBuilder {
     work_dir: PathBuf,
-    db_factory: DynDbFactory,
-    gateway_api: Url,
 }
 
 impl StandardGatewayClientBuilder {
-    pub fn new(work_dir: PathBuf, db_factory: DynDbFactory, gateway_api: Url) -> Self {
-        Self {
-            work_dir,
-            db_factory,
-            gateway_api,
-        }
+    pub fn new(work_dir: PathBuf) -> Self {
+        Self { work_dir }
     }
 }
 
 #[async_trait]
 impl IGatewayClientBuilder for StandardGatewayClientBuilder {
```

Comment:
Is it worth having a trait with only 1 impl?

Reply:
I can remove